### PR TITLE
Feature/scheduled events emails filter

### PR DESCRIPTION
**Pull request by yevhen-turovets**
The tasks in Trello 
- https://trello.com/c/AId9jK7b
- https://trello.com/c/X8WpivY3
- https://trello.com/c/vXg1ZxUW
- https://trello.com/c/bw8JPq6O
- https://trello.com/c/iLjE4lFT (FIX)

### CODE REVIEW

**Review by andytrue7 — APPROVED** [review]

File: client/src/components/scheduled-events/Past/Filter/InviteeEmails.vue [hunk 1]
```diff
@@ -88,28 +135,163 @@
...
+                return isNaN(number) ? item : number;
+            });
         }
     }
 };
 </script>
+
+<style scoped>
+.no-items {
+    color: #2c2c2c;
```
Comment (mv-v-7) [inline client/src/components/scheduled-events/Past/Filter/InviteeEmails.vue:292]: hard coded color constant

File: client/src/components/scheduled-events/Past/Filter/InviteeEmails.vue [hunk 2]
```diff
@@ -32,38 +38,76 @@
                                 hide-details
                                 dense
                                 flat
+                                v-model="searchString"
                                 color="#2C2C2C"
                                 background-color="rgba(224, 224, 224, 0.3)"
```
Comment (mv-v-7) [inline client/src/components/scheduled-events/Past/Filter/InviteeEmails.vue:43]: hardcoded color constant

File: client/src/components/scheduled-events/Past/Filter/InviteeEmails.vue [hunk 3]
```diff
@@ -32,38 +38,76 @@
                                 hide-details
                                 dense
                                 flat
+                                v-model="searchString"
```
Comment (mv-v-7) [inline client/src/components/scheduled-events/Past/Filter/InviteeEmails.vue:41]: v-model in production

**Review by mv-v-7 — CHANGES_REQUESTED** [review]

File: client/src/components/scheduled-events/Past/Filter/InviteeEmails.vue (hunk 1 above)
Comment (yevhen-turovets) [reply client/src/components/scheduled-events/Past/Filter/InviteeEmails.vue:293]: no this color in constatnt

File: client/src/components/scheduled-events/Past/Filter/InviteeEmails.vue (hunk 3 above)
Comment (yevhen-turovets) [reply client/src/components/scheduled-events/Past/Filter/InviteeEmails.vue:41]: no this color in constatnt

File: client/src/components/scheduled-events/Past/Filter/InviteeEmails.vue (hunk 2 above)
Comment (yevhen-turovets) [reply client/src/components/scheduled-events/Past/Filter/InviteeEmails.vue]: no this color in constant

**Review by aLLeXUs — APPROVED** [review]

**Pull request merged**